Signal/src/call/UserInterface/CallUIAdapter.swift

```diff
@@ -39,6 +40,7 @@ class CallUIAdapter {
     let TAG = "[CallUIAdapter]"
     private let adaptee: CallUIAdaptee
     private let contactsManager: OWSContactsManager
+    private let audioService: CallAudioService
```

Comment:
Could we eliminate the `hasManualRinger` property and push `audioService` into `NonCallKitCallUIAdaptee`? Then, `CallKitCallUIAdaptee` wouldn't even instantiate a `CallAudioService`. This way is fine too, just wondering.